eq_translations/__init__.py

```diff
@@ -3,4 +3,4 @@
 
 __all__ = ("SurveySchema", "SchemaTranslation")
 
-__version__ = "4.6.1"
+__version__ = "4.7.0"
```

Comment:
This line was added by @petechd, I'm not sure why. This should be done in setup.py

Reply:
Think it's cause that's how we used to use it: https://github.com/ONSdigital/eq-translations/pull/73/files? So this can be removed?

Reply:
It was changed in https://github.com/ONSdigital/eq-translations/pull/78/files. So I think we should revert.

Reply:
Ah I see, ok I will update

Reply:
Updated now

Reply:
I re-introduced it for the purpose of pulling by releases.

Reply:
Not sure I follow.